.github/workflows/ci.yaml

```diff
@@ -0,0 +1,31 @@
+on: [push, pull_request]
```

Comment:
When it comes to lists, I'm always a fan of the multi line version
```suggestion
on:
  - push
  - pull_request
```

The same is even more important with `python-version` in the matrix , just so you can easily add or remove entries